src/server/project.ts

```diff
@@ -2146,6 +2152,16 @@ namespace ts.server {
         getSymlinkCache() {
             return this.hostProject.getSymlinkCache();
         }
+
+        /*@internal*/
+        resolveModuleNames(moduleNames: string[], containingFile: string, reusedNames?: string[], redirectedReference?: ResolvedProjectReference, _options?: CompilerOptions, containingSourceFile?: SourceFile) {
+            return this.hostProject.resolveModuleNames(moduleNames, containingFile, reusedNames, redirectedReference, _options, containingSourceFile);

```

Comment:
I don’t think you can safely use this without changing how files / directories are watched for resolutions and failed lookups.., their lifetime, etc

Reply:
I didn’t end up needing this after all—I just wanted to make the AutoImportProviderProject and its host Project share the same module resolution cache (they necessarily share the same resolution settings).